src/server/index.ts

```diff
@@ -3,179 +3,21 @@ import dotenv from 'dotenv';
 
 checkEnvVariables(envVars);
 
+const app = express();
 const port = getEnvVariable('PORT', envVars) as number;
 
-app.get('/ready', (_, res) => {
-  res.send('Faucet backend is ready.');
-});
-
-app.get('/health', (_, res) => {
-  res.send('Faucet backend is healthy.');
-});
-
-// prometheus metrics
-app.get('/metrics', (_, res) => {
-  const errors_total = getMetrics(
-    'errors_total',
-    'counter',
-    errorCounter.total(),
-    'The total amount of errors logged on the faucet backend'
-  );
-  const errors_rpc_timeout = getMetrics(
-    'errors_rpc_timeout',
-    'counter',
-    errorCounter.getValue('rpcTimeout'),
-    'The total amount of timeout errors between the faucet backend and the rpc node'
-  );
-
-  const balance = getMetrics(
-    'balance',
-    'gauge',
-    actions.getFaucetBalance(),
-    'Current balance of the faucet',
-    true
-  );
-
-  const total_requests = getMetrics(
-    'total_requests',
-    'gauge',
-    metrics.data.total_requests,
-    'Total number of requests to the faucet'
-  );
-  const successful_requests = getMetrics(
-    'successful_requests',
-    'gauge',
-    metrics.data.success_requests,
-    'The total number of successful requests to the faucet'
-  );
-
-  res.end(
-    `${errors_total}${errors_rpc_timeout}${balance}${total_requests}${successful_requests}`
-  );
-});
-
-const createAndApplyActions = (): void => {
-  app.get<unknown, BalanceResponse>('/balance', (_, res) => {
-    actions
-      .getBalance()
-      .then((balance) => res.send({ balance }))
-      .catch((e) => {
-        logger.error(e);
-        errorCounter.plusOne('other');
-      });
-  });
-
-  app.post<unknown, DripResponse, BotRequestType>(
-    '/bot-endpoint',
-    (req, res) => {
-      const { address, amount, sender } = req.body;
-      metrics.data.total_requests++;
-
-      storage
-        .isValid(sender, address)
-        .then(async (isAllowed) => {
-          const isPrivileged =
-            sender.endsWith(':matrix.parity.io') ||
-            sender.endsWith(':web3.foundation');
-
-          // parity member have unlimited access :)
-          if (!isAllowed && !isPrivileged) {
-            res.send({
-              error: `${sender} has reached their daily quota. Only request once per day.`,
-            });
-          } else {
-            const sendTokensResult = await actions.sendTokens(address, amount);
-
-            // hash is null if something wrong happened
-            if (isDripSuccessResponse(sendTokensResult)) {
-              metrics.data.success_requests++;
-              storage.saveData(sender, address).catch((e) => {
-                logger.error(e);
-                errorCounter.plusOne('other');
-              });
-            }
-
-            res.send(sendTokensResult);
-          }
-        })
-        .catch((e) => {
-          logger.error(e);
-          errorCounter.plusOne('other');
-        });
-    }
-  );
-};
+app.use(bodyParser.json());
+app.use('/', router);
```

Comment:
These routes are now split up into separate routing files, this is best practice with express and organisation